spec/models/event_spec.rb

```diff
@@ -1,26 +1,65 @@
 require 'rails_helper'
-
 describe "Event model", type: :model do
 
-    it "should not validate without name" do
-        league = FactoryBot.build(:league, name: nil)
-        league.name = nil
+    let(:event) {FactoryBot.build(:event)}
 
-        expect(league.valid?).to eq(false)
+    it "should not validate without name" do
+      league = FactoryBot.build(:league, name: nil)
+      expect(league.valid?).to eq(false)
     end
 
     it "should not validate without discipline" do
-        league = FactoryBot.build(:league, discipline: nil)
-        league.discipline = nil
-
-        expect(league.valid?).to eq(false)
+      league = FactoryBot.build(:league, discipline: nil)
+      expect(league.valid?).to eq(false)
     end
 
     it "should not validate without game_mode" do
-        league = FactoryBot.build(:league, game_mode: nil)
-        league.game_mode = nil
+      league = FactoryBot.build(:league, game_mode: nil)
+      expect(league.valid?).to eq(false)
+    end
+
+    it "should have an attribute deadline" do
+      date = Date.new(2017,11,16)
```

Comment:
Is it a good idea to use the hard-coded date from our factory here again? What happens if we change the factory?

Reply:
This is merged code, basically copied from constantin I would've written the test differently anyways but I think it is fine for now.